src/dropdown/dropdown.component.ts

```diff
@@ -198,33 +213,37 @@ export class Dropdown implements OnInit, AfterContentInit, OnDestroy {
 	 * The `type` property specifies whether the `Dropdown` allows single selection or multi selection.
 	 */
 	ngOnInit() {
-		this.view.type = this.type;
+		if (this.view) {
+			this.view.type = this.type;
+		}
 	}
 
 	/**
 	 * Initializes classes and subscribes to events for single or multi selection.
 	 */
 	ngAfterContentInit() {
-		this.view.type = this.type;
-		this.view.size = this.size;
-		this.view.select.subscribe(event => {
-			if (this.type === "multi") {
-				this.propagateChange(this.view.getSelected());
-			} else {
-				this.closeMenu();
-				this.dropdownButton.nativeElement.focus();
-				if (event.item && event.item.selected) {
-					if (this.value) {
-						this.propagateChange(event.item[this.value]);
+		if (this.view) {
```

Comment:
It's more readable to put a guard than wrapping the whole content of the function.
 ```suggestion
		if (!this.view) {
			return;
		}
```
and then have the rest of the function unchanged/as it was before this PR.